app/api/api_v1/endpoints/test_run_executions.py

```diff
@@ -72,22 +72,11 @@ def create_test_run_execution(
     *,
     db: Session = Depends(get_db),
     test_run_execution_in: schemas.TestRunExecutionCreate,
-    selected_tests: Optional[schemas.TestSelection] = None,
+    selected_tests: schemas.SelectedTests,
```

Comment:
Could we put `SelectedTests` directly on `TestRunExecutionCreate`?

Reply:
Should we even store this original selection as a JSON blob directly? This would help "repeat" too.

I know it's a bit duplicate info. 

I am even wondering if we should move this code, To the start of the TestRunner.(We should do that in a separate PR)
https://github.com/antonio-amjr/certification-tool-backend/blob/4a2af43163aa11b0c9275b91fdcb51a45920701c/app/crud/crud_test_run_execution.py#L188-L194

Reply:
As soon as https://github.com/project-chip/certification-tool-backend/pull/6 is merged, I plan to tackle this.

Reply:
So Mikael, about this:
The `TestRunExecutionCreate` schema is used directly in the creation of the `TestRunExecution` model. The `base_class` verifies if the model has all the keys provided and gives an error for extra ones. So would be the case to include this info in the model as well?

I wasn't sure about it, so I created for now a aux crud create that receives the `selected_tests`.
Let me know what's your take on this.

Reply:
I created another issue to track this discussion and solve another time:
https://github.com/project-chip/certification-tool/issues/100

Closing this up here.